vms/platformvm/txs/executor/create_chain_test.go

```diff
@@ -195,12 +193,6 @@ func TestCreateChainTxAP3FeeChange(t *testing.T) {
 			fee:           0,
 			expectedError: nil,
 		},
-		{
-			name:          "post-fork - incorrectly priced",
-			time:          ap3Time,
-			fee:           100*defaultTxFee - 1*units.NanoAvax,
-			expectedError: utxo.ErrInsufficientUnlockedFunds,
```

Comment:
because the fee is now being set to `0` we won't get this error. Could also make sense to remove these tests altogether.